### DISCUSSION TITLE
Crazyflie 2.1 will not power on at all

### DISCUSSION THREAD

**Discussion by adonaruma** [bitcraze/discussions#1219] [Q&A] [answered]
HS teacher here. We bought a set of 10 crazyflie 2.1 drones through DroneBlocks for my coding exploratory class. All of the drones worked perfect for the first couple classes, however 1 of them has completely died on me. There is no apparent damage to the drone body that I can see. Yet, whenever I plug in a battery or even use a USB connection to turn it on, it will not register at all. No LEDs will light up at all, and there appears to be some heat being generated in the connection which is very concerning. I can not get it connected to reset or update firmware through any of the methods seen on forums.

**Comment by ataffanel** [comment, marked as answer]
This looks like a hardware problem, very sorry about that, this is not something we see often. You should contact DroneBlocks for a replacement.

The failsafe way to boot a Crazyflie is to press the button while battery and USB is disconnected, and then to connect the battery while the button is still pressed (it might require 3 hands ...),the M2 LED should light up and when it blinks the button can be released, this should put both CPUs in bootloader mode and both blue led should blink. If this does not happen on your Crazyflie, there is definitly something wrong with the hardware. Likely a short near the power management chips.
